### PR TITLE
 Remove min_profile

### PR DESCRIPTION
**Pull request by limingu**
Fix #7292, #8355
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by marstr** [comment]
I believe you fell victim to https://github.com/Azure/azure-cli/issues/9648. Please rebase on top of [`dev`](https://github.com/Azure/azure-cli/tree/dev) (d2cf98f) to adopt the newest changes to CI, which should unblock your contribution.

File: src/azure-cli-core/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Release History
 ===============
 
+* Remove min_profile
```
Comment (tjprescott) [inline src/azure-cli-core/HISTORY.rst]: This needs to be more specific and annotated as a breaking change.

File: src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/__init__.py [hunk 2]
```diff
@@ -12,10 +12,11 @@ class ServiceFabricCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
+        from azure.cli.core.profiles import ResourceType
         sf_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.servicefabric.custom#{}')
         super(ServiceFabricCommandsLoader, self).__init__(cli_ctx=cli_ctx,
                                                           custom_command_type=sf_custom,
-                                                          min_profile='2017-03-10-profile')
+                                                          resource_type=ResourceType.MGMT_SEAVICEFABRIC)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/__init__.py]: Typo: This should be MGMT_SERVICEFABRIC

File: src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/__init__.py [hunk 3]
```diff
@@ -15,9 +15,10 @@ class ServicebusCommandsLoader(AzCommandsLoader):
     def __init__(self, cli_ctx=None):
         from azure.cli.core import ModExtensionSuppress
         from azure.cli.core.commands import CliCommandType
+        from azure.cli.core.profiles import ResourceType
         servicebus_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.servicebus.custom#{}')
         super(ServicebusCommandsLoader, self).__init__(cli_ctx=cli_ctx, custom_command_type=servicebus_custom,
-                                                       min_profile='2017-03-10-profile',
+                                                       resource_type=ResourceType.MGMT_SEAVICEBUS,
```
Comment (tjprescott) [inline src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/__init__.py]: Typo: MGMT_SERVICEBUS

File: src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/__init__.py [hunk 4]
```diff
@@ -32,11 +32,12 @@ def handler(ctx, **kwargs):
 class IoTCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
+        from azure.cli.core.profiles import ResourceType
         iot_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.iot.custom#{}')
         cli_ctx.register_event(EVENT_INVOKER_POST_PARSE_ARGS, handler)
         super(IoTCommandsLoader, self).__init__(cli_ctx=cli_ctx,
                                                 custom_command_type=iot_custom,
-                                                min_profile='2017-03-10-profile')
+                                                resource_type=ResourceType.MGMT_IOTHUB)
```
Comment (tjprescott) [inline src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/__init__.py]: I believe this should just be MGMT_IOT.

File: scripts/ci/test_extensions.sh [hunk 5]
```diff
@@ -29,11 +29,12 @@ for ext in $output; do
...
-    exit_code=1
-    echo "Failed to verify:" $ext
-fi
+# Skip the test for now. It will be enable after release 2.0.67
+#azdev verify load-all
+#if [ $? != 0 ]
+#then
+#    exit_code=1
+#    echo "Failed to verify:" $ext
+#fi
```
Comment (tjprescott) [inline scripts/ci/test_extensions.sh]: Does this still need to be commented out?

**Comment by tjprescott** [comment]
Also which specific issues is this PR resolving?

File: scripts/ci/test_extensions.sh (hunk 5 above)
Comment (limingu) [reply scripts/ci/test_extensions.sh]: Yes, we still need it as the iot extension didn't publish the new version with max cli version. Iot extension will fail the azdev verify load-all test.

**Comment by tjprescott** [comment]
I want to wait until this PR is merged to fork the release branch. This needs to be in the dev branch so we can update extensions between now and Tuesday.

**Comment by limingu** [comment]
> Also which specific issues is this PR resolving?

Fix #7279, #8355

**Pull request merged**